packages/wrangler/src/config/environment.ts

```diff
@@ -484,6 +484,15 @@ interface EnvironmentNonInheritable {
 		dataset?: string;
 	}[];
 
+	/**
+	 * A browser name that will be usable from the worker.
+	 */
+	browser:
```

Comment:
Pretty much every other binding type supports an array, for multiple linked bindings on a worker. Is there a reason only one browser is allowed?

Reply:
I replied to this in another comment.